bindings/go/blob/go.mod

```diff
@@ -5,15 +5,24 @@ go 1.24.1
 require (
 	github.com/opencontainers/go-digest v1.0.0
 	github.com/stretchr/testify v1.10.0
+	ocm.software/open-component-model/bindings/go/runtime v0.0.2
```

Comment:
Hmm i really dont know if we want the runtime to be part of the dependency chain for blobs

Reply:
resolved. we try a separate module and if that doesnt work we go for package inclusion